.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
```

Comment:
Can you add a hook for black as well?

Reply:
Added, also it turns out that isort and black don't agree with each other.